tests/test_langchain_tools_vcr.py

```
@@ -148,14 +150,19 @@ def test_json_to_pptx_analyze_tool_vcr(client: ToolsClient) -> None:
     """
+    # 環境変数が設定されていない場合は、ダミーのUUIDを使用
     template_id = (
-        os.getenv("MIDDLEMAN_TEST_TEMPLATE_ID") or ""
+        os.getenv("MIDDLEMAN_TEST_TEMPLATE_ID") or "00000000-0000-0000-0000-000000000000"
     )  # テスト用のテンプレートID
     tool = JsonToPptxAnalyzeTool(client=client)
-    result = tool._run(template_id)
-
-    assert isinstance(result, str)
-    assert "Slide" in result
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)